components/admin/lessons/AdminLessonExerciseCard.tsx

```diff
@@ -0,0 +1,167 @@
+import { get, toUpper } from 'lodash'
+import React, { useState } from 'react'
+import { Collapse, Spinner } from 'react-bootstrap'
+import {
+  Exercise,
+  useDeleteExerciseMutation,
+  User,
+  useRemoveExerciseFlagMutation
+} from '../../../graphql'
+import styles from '../../../scss/adminLessonExerciseCard.module.scss'
+import CopyButton from '../../CopyButton'
+
+type HeaderProps = { user: User; exercise: Exercise }
+const Header = ({ user, exercise }: HeaderProps) => {
+  return (
+    <div className={styles.card__header}>
+      <div className={styles.card__badges}>
+        {exercise.flaggedAt && (
+          <span
+            className={`${styles.card__header__badge} ${styles.card__header__badge__flagged}`}
+          >
+            FLAGGED
+          </span>
+        )}
+        <span
+          className={`${styles.card__header__badge} ${styles.card__header__badge__module}`}
+        >
+          {toUpper(exercise.module.name)}
```

Comment:
nit: since this is purely for appearance we shoul consider doing such modifications using the css text-transform property instead of javascript.